### DISCUSSION TITLE
Disable Snow Golem Trails

### DISCUSSION THREAD

**Discussion by blenditbeautiful** [GriefPrevention/GriefPrevention#595] [Ideas]
As administrator I'd like to see a toggle added to the config to disable snow golem trails either globally or within claims as griefers can and do use them to create a mess.

Thanks for your time.

**Comment by RoboMWM** [comment]
Players use them as "snow" farms, so however this is done, it probably shouldn't be a default. Not to mention it's preventable with light sources.

**Comment by blenditbeautiful** [comment]
Perhaps it should be like surface explosions? Players can enable it within their claim for their own purposes but otherwise they don't leave trails.
